### ISSUE TITLE
[Feature Request]  Add same color multiple times,  when using in combination 

### ISSUE THREAD

**Issue by shadowbone94** [glesik/wpgen#3]
So by that I mean while setting a combination of wallpaper, for example 'black' and 'yellow', I can only set the output wallpaper as 'black+yellow'.
Instead if it is possible to make it as 'black+yellow+black+yellow' or 'black+yellow+black' etc..
In other words, the same colour multiple times. Because now trying in combination, when I select a color the second time,  it gets deselected.

**Comment by glesik** [comment]
Fair enough, I haven't thought of this scenario. That would require totally different selection method. A "plus" button on each color? Add to stack on long press? Do you know any standard design pattern for such case?

Meanwhile, as a workaround one can add slightly different color, e.g. #000001 for "almost black" in your case.

**Comment by shadowbone94** [comment]
@glesik
The work around you suggested is working alright,  more or less. 

But if its worth the effort, do try and put a panel box above the platelets. 
That way, when a colour platelet is selected (eg: Blue), the first box in the panel goes blue.  When the 2nd platelet is selected (say red) , the 2nd box of the panel gets the red colour selected and so on... 
(I hope you got the gist. Somewhat like how collage apps work)

But then again if its not worth it,  needn't bother. Because the alternative you suggested is almost fine by me. :smile:
